controller/sync.go

```diff
@@ -68,9 +82,23 @@ func (s *SubgraphSyncer) checkNewSubgraphSync() (bool, error) {
 	return false, nil
 }
 
+func (s *SubgraphSyncer) historicalSyncCandles(notif chan bool) {
+	notif <- true // Signal that the server is ready to accept requests
+	// UNISWAP_CANDLE_LOOKBACK_WINDOW :=  int(time.Now().Unix()) -  3600 *24 *7
+	JAN_1_2023_GMT :=  1674259200 
+	initialSyncTime := 	JAN_1_2023_GMT
```

Comment:
Would it make sence to put this into the .env file so that its not hardcoded here? just incase its ever wanted to be changed.